### DISCUSSION TITLE
convert_to_pdf throw an error: pdf device does not support substitute fonts

### DISCUSSION THREAD

**Discussion by hbh112233abc** [pymupdf/PyMuPDF#1258] [General]
hi,

I use `convert_to_pdf` to copy some pdf,but the below file throw an error:  ==pdf device does not support substitute fonts==

[dwg.pdf](https://github.com/pymupdf/PyMuPDF/files/7150986/dwg.pdf)

my env:
win10 64bit
python 3.7.3
pymupdf 1.18.17

Do you have some solution for this problem??

**Comment by JorjMcKie** [comment]
No, sorry. PDF-to-PDF conversion does not work under all circumstances. Most often, use of unsupported font types is the reason.
